src/html.jl

```diff
@@ -260,8 +261,10 @@ function fix_links(names, pages, url_prefix)
             elseif startswith(capture, "#ref-")
                 page_link = "references"
                 return uncapture("$url_prefix/$page_link.html$capture")
-            else
+            elseif URIs.URI(URIs.unescapeuri(capture)).scheme == ""
```

Comment:
The change I make here is to ensure that we only add the `url_prefix` when the scheme part is undefined, in which case the `capture` is usually a relative or absolute path.